panel/models/reactive_html.py

```diff
@@ -106,6 +106,7 @@ def handle_data(self, data):
         values_loop = re.findall(values_iter_re, data)
         items_loop = re.findall(items_iter_re, data)
         nloops = len(list_loop) + len(values_loop) + len(items_loop)
+        print(list_loop)
```

Comment:
Loose print statement. Wondering if pre-commit could/should catch a print statement like this?

Reply:
I've seen a pre-commit plugin that does that. There are a few other print statements in the code base, so first we'd have to check if they can be replaced by logging calls, and replace them if so.

Reply:
Thanks @Hoxbro. Can also mark them explicitly to be ignored.